ArduPlane/Parameters.h

```diff
@@ -128,7 +128,7 @@ class Parameters {
         k_param_override_channel,
         k_param_stall_prevention,
         k_param_optflow,
-        k_param_cli_enabled_old, // unused - CLI removed
```

Comment:
This is brave.  I mean, yeah, unused for years, but re-using may lead to arguments.